native-schema-registry/csharp/AWSGsrSerDe/AWSGsrSerDe.Tests/User.cs

```diff
@@ -0,0 +1,43 @@
+using Avro;
+using Avro.Specific;
+
+namespace AWSGsrSerDe.Tests
+{
+    public class User : ISpecificRecord
+    {
+        
+        private static Schema _schema = Schema.Parse("{\"type\":\"record\",\"name\":\"User\",\"namespace\":\"AWSGsrSerDe.Tests\",\"fields\":[{\"name\":\"name\",\"type\":\"string\"},{\"name\":\"favorite_number\",\"type\":[\"int\",\"null\"]},{\"name\":\"favorite_color\",\"type\":[\"string\",\"null\"]}]}");
+
+        private string _name;
+        private int _favoriteNumber;
+        private string _favoriteColor;
+        
+
+        public object Get(int fieldPos)
```

Comment:
This class is used to construct Avro SpecificRecord and this method must be implemented since it is part of the interface. I will add additional getters/setters.